http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -860,6 +868,8 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
       autoscalingConfig = None
     )
 
+    // regionName can be empty in some test configurations
+    val regionName = if (azureRegionOpt.isEmpty) "" else azureRegionOpt.map(_.name).getOrElse("")
```

Comment:
is `Option.isEmpty` true if the value of the option is `None`?

Reply:
Yes, I tested it as I wasn't sure either.